README.md

```diff
@@ -1,17 +1,3 @@
 # python-learning
 
-This repository is for python beginners.
-
-## This is second heading
-
-### This is 3rd heading
-
-#### This is 4th heading
-
-#####This is 5th
-
-######This is 6th
-only 6 heading are possible
-
-* Hello there
-    * what are you doing?
\ No newline at end of file
+This repository is for python beginners.
```

Comment:
You could have added more description about python problems etc. But, this looks good for now. Thanks for doing this @manumanasa.